src/data_gradients/feature_extractors/classification/class_distribution.py

```diff
@@ -0,0 +1,79 @@
+import collections
+
+import pandas as pd
+
+from data_gradients.common.registry.registry import register_feature_extractor
+from data_gradients.feature_extractors.abstract_feature_extractor import Feature
+from data_gradients.utils.data_classes.data_samples import ClassificationSample
+from data_gradients.visualize.seaborn_renderer import BarPlotOptions
+from data_gradients.feature_extractors.abstract_feature_extractor import AbstractFeatureExtractor
+
+
+@register_feature_extractor()
+class ClassificationClassDistribution(AbstractFeatureExtractor):
+    """Feature Extractor to count the number of labels of each class."""
+
+    def __init__(self):
+        self.data = []
+
+    def update(self, sample: ClassificationSample):
+        class_name = sample.class_names[sample.class_id]
+        self.data.append(
+            {
+                "split": sample.split,
+                "class_id": sample.class_id,
+                "class_name": class_name,
+            }
+        )
+
+    def aggregate(self) -> Feature:
+        df = pd.DataFrame(self.data)
+
+        # Include ("class_name", "split", "n_appearance")
+        df_class_count = df.groupby(["class_name", "class_id", "split"]).size().reset_index(name="n_appearance")
+        all_class_names = df_class_count["class_name"].unique()
+
+        # Height of the plot is proportional to the number of classes
+        n_unique = len(all_class_names)
+        figsize_x = 10
+        figsize_y = min(max(6, int(n_unique * 0.3)), 175)
```

Comment:
In segmentation and classification, we have an option to filter to only keep topk

```python
df_class_count = self.value_extractor.select(df=df_class_count, id_col="class_id", split_col="split", value_col="frequency")
```

I think it would be good to add this option (great for cases with huge number of classes)

Reply:
Fixed in 8530cb21cca2851d3fadfdc0f57f82362db43853 as well.
However top_k argument is made None by default. 
I beleive what we eventually want to do is have top_k is argument in analysis manager.
So user can tune top_k from the API.